R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -2253,6 +2253,15 @@ test_that("mutate(), transform(), rename() and names()", {
   detach(airquality)
 })
 
+test_that("alias on SparkDataFrame", {
+  df <- alias(read.df(jsonPath, "json"), "table")
```

Comment:
because trying to make a set of tests that makes sense for CRAN
https://github.com/apache/spark/pull/17817